sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -55,6 +56,26 @@ def test_mean_shift():
     assert_equal(n_clusters_, n_clusters)
 
 
+def test_mean_shift_negative_bandwidth():
+    bandwidth = -1
+    ms = MeanShift(bandwidth=bandwidth)
+    msg = ("bandwidth needs to be greater than zero or None,"
+           "            got -1.000000")
```

Comment:
This whitespace looks like an error in the code raising the message. Please change the code to have a single space between the comma and "got"

Reply:
This is unresolved. Please fix the error message in mean_shift_.py